layers/meta-opentrons/recipes-robot/opentrons-robot-app/files/opentrons-robot-app.service.in

```diff
@@ -5,10 +5,13 @@ Requires=weston@root.service
 
 [Service]
 Restart=on-failure
-Type=forking
+Type=notify
 Environment=@@application-environment@@
 ExecStart=/usr/bin/opentrons-robot-app.sh
 RestartSec=1
+TimeoutStartSec=30
```

Comment:
is this enough startup time?

Reply:
don't really know, honestly. i'll be messing with it to verify.

Reply:
Yes it is!